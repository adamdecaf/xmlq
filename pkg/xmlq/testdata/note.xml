<?xml version="1.0"?>
<note>
  <to>Tove</to>
  <from>Jani</from>
  <heading>Reminder</heading>
  <body>Don't forget me this weekend!</body>
  <extra>  Preserve spaces on both sides  </extra>
  <additional>5 > 3</additional>
</note>
