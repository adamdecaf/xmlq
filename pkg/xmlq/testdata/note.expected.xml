<?xml version="1.0"?>
<note>
  <to>Tove</to>
  <from>*an*</from>
  <heading>Reminder</heading>
  <body>Don&#39;t forget me this weekend!</body>
</note>
